tests/util.go

```diff
@@ -29,7 +29,10 @@ type Result struct {
 	params        []string
 }
 
-const NamespaceTestDefault = "kubevirt-test-default"
+const (
+	NamespaceTestDefault  = "kubevirt-test-default"
```

Comment:
This constant seems familiar to me, it is defined in kubevirt repo.

Reply:
Please remove this one.

Reply:
this one is not introduced in this PR, I'm afraid other tests are using it. It need an overhaul to remove it.

Reply:
Hm ... I see now ...